protoyaml_test.go

```diff
@@ -243,7 +242,6 @@ func TestInfNanIntegers(t *testing.T) {
 		"single_int64: -Infinity",
 		"single_int64: NaN",
 	} {
-		testCase := testCase
```

Comment:
Is this the reason you updated to Go 1.22 -- to get the thread-safe for loops?

Reply:
the linter complained. I guess the linter needs to run at the lowest version instead of the highest version?

Reply:
Hmm not quite sure what happened: https://github.com/bufbuild/protoyaml-go/actions/runs/10320536754/job/28571346098
But it works now.